modules/ROOT/nav.adoc

```diff
@@ -1,10 +1,13 @@
 .xref:index.adoc[Anypoint Visualizer]
```

Comment:
KIM
-Manage Performance and Data Collection (on each changed file)
-Enable Header Injection (standalone-mule-setup, runtime-fabric-setup; the instructions are different)
-Select a Deployment Type (setup)
-Reduce Potential Performance Impact for CloudHub/Standalone Mule/Runtime Fabric (each heading is in technical.adoc)